hw/xbox/nv2a/nv2a.h

```diff
@@ -248,7 +247,8 @@ typedef struct PGRAPHState {
     bool texture_matrix_enable[NV2A_MAX_TEXTURES];
 
     /* FIXME: Move to NV_PGRAPH_BUMPMAT... */
-    float bump_env_matrix[NV2A_MAX_TEXTURES-1][4]; /* 3 allowed stages with 2x2 matrix each */
+    float bump_env_matrix[NV2A_MAX_TEXTURES - 1][4]; /* 3 allowed stages with
+                                                        2x2 matrix each */
```

Comment:
I dislike how the comment is split so weirdly. Even if I was on a 80 column editor, I'd rather have my terminal split this.